sdk/spring/azure-spring-boot-test-b2c/src/test/java/com/azure/test/b2c/utils/B2CTestUtils.java

```diff
@@ -0,0 +1,13 @@
+package com.azure.test.b2c.utils;
+
+public class B2CTestUtils {
+
+    public static final String B2C_USER_EMAIL = "chrisgreen@abc.com";
+    public static final String B2C_USER_PASSWORD = "B2C_USER_PASSWORD";
+    public static final String B2C_TENANT = "B2CITTest1";
+    public static final String B2C_CLIENT_ID = "B2C_CLIENT_ID";
+    public static final String B2C_CLIENT_SECRET = "B2C_CLIENT_SECRET";
+    public static final String B2C_REPLY_URL = "http://localhost:8080/home";
+    public static final String B2C_SIGN_UP_OR_SIGN_IN = "B2C_1_signupsignin1";
+    public static final String B2C_PROFILE_EDIT = "B2C_1_profileediting1";
+}
```

Comment:
Please remove duplicated files of `chromedeiver_xxx`.

Reply:
This will be removed in the next PR.Use a public `chromedeiver_xxx`.